File: TOMIMSV4/Entity/Mantenimientos/Unidad_Medida_Conversion/clsBeUnidad_medida_conversion.vb
Public Class clsBeUnidad_medida_conversion
    Implements ICloneable
    Public Property Activo As Boolean
    Public Property IdConversion() As Integer = 0
    Public Property IdUnidadMedidaOrigen() As Integer = 0
    Public Property IdUnidadMedidaDestino() As Integer = 0
    Public Property Factor() As Double = 0.0

    Sub New()
    End Sub

    Sub New(ByRef IdConversion As Integer, ByVal IdUnidadMedidaOrigen As Integer, ByVal IdUnidadMedidaDestino As Integer, ByVal Factor As Double, ByVal Activo As Boolean)
        Me.IdConversion = IdConversion
        Me.IdUnidadMedidaOrigen = IdUnidadMedidaOrigen
        Me.IdUnidadMedidaDestino = IdUnidadMedidaDestino
        Me.Factor = Factor
        Activo = Activo
    End Sub

    Public Function Clone() As Object Implements System.ICloneable.Clone
        Return MyBase.MemberwiseClone()
    End Function

End Class

File: WMS.EntityCore/Datos_Maestros/clsBeUnidad_medida_conversion.cs
using System.ComponentModel;
using System.ComponentModel.DataAnnotations.Schema;

namespace WMSWebAPI.Be
{
    public class clsBeUnidad_medida_conversion : ICloneable
    {
        [Column("IdConversion")]
        [DisplayName("IdConversion")]
        public int IdConversion { get; set; } = 0;

        [Column("IdUnidadMedidaOrigen")]
        [DisplayName("IdUnidadMedidaOrigen")]
        public int IdUnidadMedidaOrigen { get; set; } = 0;

        [Column("IdUnidadMedidaDestino")]
        [DisplayName("IdUnidadMedidaDestino")]
        public int IdUnidadMedidaDestino { get; set; } = 0;

        [Column("Factor")]
        [DisplayName("Factor")]
        public double Factor { get; set; } = 0;

        [Column("activo")]
        [DisplayName("activo")]
        public bool Activo { get; set; } = false;

        public clsBeUnidad_medida_conversion() { }

        public object Clone()
        {
            return this.MemberwiseClone();
        }
    }
}
